packages/search/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
 ## 1.0.0
 
 - Add Search component based on `calypso/components/search`
+
+## 1.0.1
+
+- Add search modes
+- Add "when-typing" search mode
+- Add "on-enter" searchmode.
```

Comment:
Can be consolidated to one 'Add "on-enter" and "when-typing" search modes.' entry.